utils/utils.go

```diff
@@ -340,3 +340,20 @@ func GetLabelOptions(selinuxOptions *pb.SELinuxOption) []string {
 	}
 	return labels
 }
+
+// Unmount is a helper function that unmounts
+// a path, accounting for symlinks
+func Unmount(path string) error {
+	// we may get files that are symlinked
+	// so we first resolve the symlink
+	fp, err := securejoin.SecureJoin("/", path)
+	if err != nil {
+		return err
+	}
```

Comment:
is it necessary?  Unmount should already resolve symlinks

Reply:
one would think so, but it was there before... I'd be fine removing